Lib/test/test_zipfile.py

```diff
@@ -616,6 +616,18 @@ def test_add_file_after_2107(self):
         except OverflowError:
             self.skipTest('Host fs cannot set timestamp to required value.')
 
+        mtime_ns = os.stat(TESTFN).st_mtime_ns
+        if mtime_ns != (4386268800 * 10**9):
```

Comment:
Just curious. Would you consider this better or worse than the 2 lines one?

```suggestion
        if (mtime_ns := os.stat(TESTFN).st_mtime_ns) != (4386268800 * 10**9):
```

Reply:
My opinion: I prefer to not use "mtime := ..." here. I like to have an explicitement assignement statement on a separated line ;-)